timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/rdf/Collection.java

```diff
@@ -96,8 +109,35 @@ public void addProperty(Vertex entityVertex, String propName, String value) {
 
     Iterator<Vertex> vertices = vertex.vertices(Direction.OUT, HAS_PROPERTY_RELATION_NAME);
 
-    addNewPropertyConfig(propName, collectionPropertyName, vertices);
+    addNewPropertyConfig(propName, collectionPropertyName, vertices,
+      new StringToStringConverter().getUniqueTypeIdentifier());
+  }
+
+  public void addToListProperty(Vertex entityVertex, String propName, String value) {
```

Comment:
ik vind deze code eigenlijk in de Altlabelprocessor horen. Maar misschien kan dat niet?